docs/DeveloperGuide.md

```diff
@@ -362,35 +370,40 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 2. HelloFile replaces the tag of the file with the new tag.
 
     Use case ends.
-    
+        
 **Extensions**
 
 * 2a. HelloFile failed to recognize the tag.
 
 	* 2a1. HelloFile prompts that tag does not exist.
 	
     	Use case resumes from step 1
-    	
+    	    	
 * 2b. HelloFile failed to recognize the new tag input.
 
     * 2b1. HelloFile prompts that the new tag is invalid.
-    
-        Use case resumes from step 1. 
         
+        Use case resumes from step 1. 
+                
 * 2c. HelloFile detects a duplicate tag name.
 
 	* 2c1. HelloFile prompts the user for another tag name. 
-	
+		
         Use case resumes from step 1.
-        
+                
 * *a. HelloFile has no permission to read or write to the file system.
 
 	* *a1. HelloFile prompts the user that it needs the read and write permission.
-	
+		
 	    Use case resumes.
+	    
+	    
+<br />
+
 
 <br />
```

Comment:
Should there be 2 br ?

Reply:
not sure

Reply:
I think between other points it's 1 br, but not a big problem